node/config/def.go

```diff
@@ -107,6 +107,9 @@ func DefaultFullNode() *FullNode {
 			MaxFilterResults:        10000,
 			MaxFilterHeightRange:    2880, // conservative limit of one day
 		},
+		EthTxHashConfig: EthTxHashConfig{
+			EnableEthHashToFilecoinCidMapping: true,
```

Comment:
I'd prefer this be disabled by default.

Initially majority of nodes won't care about eth transactions, and this is another thing in the node repo which right now will grow in an unbounded way.

Reply:
Yeah, I'll change this. Just had it on by default for ease of testing.